### DISCUSSION TITLE
How to configure the alarm in airplay2 speakers

### DISCUSSION THREAD

**Discussion by rasazu** [MiguelRipoll23/homebridge-securitysystem#148] [Q&A] [answered]
Hi,
Any idea on how replace the siren with a sound trough an Apple Homepod?

Thanks !!

**Comment by MiguelRipoll23** [comment, marked as answer]
Hey there,

[Check this out.](https://github.com/MiguelRipoll23/homebridge-securitysystem/wiki/Playing-audio-with-AirPlay-2)

**Reply by rasazu** [reply]
Thanks !!!
I didnt know obout that automatization.

**Comment by plgrimsley** [comment]
I don't have an Airplay 2 speaker, but there is another way to create a siren sound if you have an appleTV 4 or later and a TV that you can either turn on via your home app (doesn't matter how, IR or via CEC or via Homebridge or Home Assistant). 

You can create a scene in the home app to play audio via your Apple TV, the audio source can either be via your Apple Music subscription or your iTunes/Apple Music library.

Then create an automation to trigger the scene when the siren is activated, rather than triggering 'play audio' on a HomePod.

It's best to create another scene to quickly pause the music/sound if you set the alarm off by accident.
